### ISSUE TITLE
deap not useful for multiple outputs?

### ISSUE THREAD

**Issue by bionicles** [DEAP/deap#388]
It seems impossible (or impossibly unclear from reading docs) to make a multi-output deap project... Why is it so difficult to make a GP output more than one number? 

also, it's not really clear how one would use DEAP to genetically program tensorflow graphs, which is an obvious use case for the project. 

if individuals in deap cannot handle simple task of outputting multiple values, and if we can't constrain the shape of arrays/tensors in the output, why would anyone use this project instead of Tensorflow or Pytorch?

**Comment by fmder** [comment]
_I wouldn't wan't to be rude_, but this project has been around since long before Tensorflow or Pytorch. Its last major version update predates Tensorflow by 2 years. So sorry if we didn't thought about those libraries.

As to usefulness, DEAP solves a totally different kind of problem that is blackbox optimization (gradient free). And, there are multiples ways to output multiple values with GP. It doesn't take much imagination to optimize a set of trees or use cooperative optimization to optimize multiple population of trees.

Anyway, thanks for your comments, but this is an open source project and as the license clearly state 

> [...] WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

**Comment by bionicles** [comment]
@fmder Sorry. I was frustrated when I wrote this because I really want to love DEAP and use it extensively but it feels like it takes a lot of digging to accomplish what seems like a simple thing...I checked the forums and a lot of people had similar questions about this particular issue, being ignored. 

_I wouldn't want to raise an invalid issue,_ but, do you really think it's a good use of resources for users to make 1 population of trees per scalar output, and for this issue to not be clearly covered in the docs? What if we're talking about RL tasks with many continous outputs? When we're talking about a molecular dynamics project with XYZ vector for each atom in a protein complex with 5,000-10,000 atoms, or a robotics project with many degrees of freedom, we need N trees for N different dimensions, and they have to evolve independently? 

Sounds like a ton of extra work just to avoid using a small amount of brainpower
